packages/an-delay-app/contracts/Delay.sol

```diff
@@ -46,7 +47,7 @@ contract Delay is AragonApp, IForwarder, IForwarderFee {
     uint256 public feeAmount;
     address public feeDestination;
 
-    event DelayedScriptStored(uint256 scriptId);
+    event DelayedScriptStored(uint256 scriptId, bytes evmCallScript);
```

Comment:
Could we add the `feeAmount` and `executionTime` as params too?

The only way to get this data is by calling the contract which could be in a state different than the one the contract was in when the delayed script was created